package.json

```diff
@@ -55,7 +56,8 @@
     "react-foco": "^1.3.0",
     "react-focus-lock": "^2.3.1",
     "react-overlays": "^2.1.0",
-    "react-popper": "^1.3.3",
+    "react-popper": "^2.2.3",
+    "react-popper-2": "npm:react-popper@^2.2.3",
```

Comment:
This is temporary to fix an issue with Storybook
Storybook: https://github.com/storybookjs/storybook/issues/10982

Storybook must be doing something with the how deps are resolved, but it should work after they update react-popper-tooltip, tracking that here: https://github.com/mohsinulhaq/react-popper-tooltip/issues/92